mava/configs/env/gigastep.yaml

```
@@ -0,0 +1,13 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: hide_and_seek_5_vs_5 # [hide_and_seek_5_vs_5, identical_5_vs_5, waypoint_5_vs_5]
+
+env_name: Gigastep
+
+eval_metric: win_rate
+
+kwargs:
+  time_limit: 500
+  get_adversary_obs: False
+  get_adversary_actions: False
```

Can all the settings be grouped to improve the readability? 
```suggestion
environment:
  name: Gigastep
  eval_metric: win_rate
  time_limit: 500
  get_adversary_obs: False
  get_adversary_actions: False
```

---

I tried to follow a similar structure to the configs of other environments
although changing them would improve readability i don't know if it's worth the inconsistency between the config file structures of different environments & extra logic to parse them

---

Thank you for helping with the reviewing @sabrinekr :fire: 
Indeed, this is the default pattern we follow in Mava, the reason for this is since Hydra will parse all the contents env config yaml file to a nested DictConfig. Eg in the parsed config it will currently be `env.env_name`, `env.eval_metric` and `env.kwargs`.

---

No need to place all info under an "environment", as `hydra` allows you to access these values under` config.env` (envs configs under env folder), and from there you can access the various flags and values specific to the **_selected_** environment in the [system default files](https://github.com/instadeepai/Mava/blob/49ec90c3a30f2b52fc5f4f423bb92007c0b7e367/mava/configs/default_ff_ippo.yaml#L6). (e.g in [make_env()](https://github.com/instadeepai/Mava/blob/49ec90c3a30f2b52fc5f4f423bb92007c0b7e367/mava/utils/make_env.py#L76) )